api/controllers/ApplicationTypeController.js

```diff
@@ -243,69 +248,84 @@ module.exports = {
      * @return res.view
      */
     newApplication: function (req, res) {
-        const company_name = req.session.user && (req.session.user.premiumEnabled || req.session.user.dropOffEnabled)
-            ? req.session.account.company_name
-            :'N/A';
+        const company_name =
+            req.session.user &&
+            (req.session.user.premiumEnabled || req.session.user.dropOffEnabled)
+                ? req.session.account.company_name
+                : 'N/A';
         const loggedIn = HelperService.LoggedInStatus(req);
         let selectedServiceType = req.param('app_type_group');
 
         if (typeof selectedServiceType === 'undefined') {
             selectedServiceType = 'not ok';
         }
 
-        const premiumOrDropoffPageSelected = ['2','3'].includes(selectedServiceType);
+        const premiumOrDropoffPageSelected = ['2', '3'].includes(
+            selectedServiceType
+        );
```

Comment:
`2 == 'premium type'` and `3 == 'drop off type'`?

Sorry, I'm just allergic to magic numbers.

Reply:
That's fair, it does look cleaner without magic numbers 🪄